mmv1/templates/terraform/constants/subnetwork.erb

```diff
@@ -17,3 +17,33 @@ func IsShrinkageIpCidr(_ context.Context, old, new, _ interface{}) bool {
 
 	return true
 }
+
+func sendSecondaryIpRangeIfEmptyDiff(_ context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	if diff.Id() == "" {
+		return nil
+	}
+
+	sendZero := diff.Get("send_secondary_ip_range_if_empty").(bool)
+	if !sendZero {
+		return nil
+	}
+
+	configSecondaryIpRange := diff.GetRawConfig().GetAttr("secondary_ip_range")
+	if !configSecondaryIpRange.IsKnown() {
```

Comment:
> Also flagging that the unknown nonzero value case has not been covered or I have not realised it has been covered anywhere yet! (apologies for two topics in one thread)

@rileykarson Doesn't this covered all unknown cases? Exit early if the value is not known

Reply:
Oh, yep! Could confirm the intended behaviour with a test (one-off manual would be fine) that we see the intended behavious with an unknown value for a secondary range?

Reply:
yes here it is (with some added debug logs for `IsKnown` / `isWhollyKnown`: paste/5787900929376256#l=324